src/components/__tests__/FormFieldSelectTable.cy.js

```diff
@@ -1,10 +1,27 @@
 import FormFieldSelectTable from 'components/form/FormFieldSelectTable.vue';
 import { rideToWorkByBikeConfig } from 'src/boot/global_vars';
 import { i18n } from '../../boot/i18n';
+import { useSelectOrganization } from 'src/composables/useSelectOrganization';
+import { createPinia, setActivePinia } from 'pinia';
+// import { useRegisterChallengeStore } from 'src/stores/registerChallenge';
 
 const { contactEmail } = rideToWorkByBikeConfig;
 
 describe('<FormFieldSelectTable>', () => {
+  let options;
+
+  before(() => {
+    setActivePinia(createPinia());
+    // const store = useRegisterChallengeStore();
```

Comment:
If you don't need this line remove it please.

Reply:
Removed.